sparse_strips/vello_cpu/tests/glyph.rs

```diff
@@ -98,13 +98,53 @@ fn glyphs_skewed_unhinted() {
     ctx.set_paint(REBECCA_PURPLE.with_alpha(0.5).into());
     ctx.glyph_run(&font)
         .font_size(font_size)
-        .horizontal_skew(-20_f32.to_radians())
+        .glyph_transform(Affine::skew(-20_f64.to_radians().tan(), 0.))
         .hint(false)
         .fill_glyphs(glyphs.into_iter());
 
     check_ref(&ctx, "glyphs_skewed_unhinted");
 }
 
+#[test]
+fn glyphs_skewed_unskewed() {
+    let mut ctx = get_ctx(150, 125, false);
+    let font_size: f32 = 50_f32;
+    let (font, glyphs) = layout_glyphs("Hello,\nworld!", font_size);
+
+    ctx.set_transform(
+        Affine::translate((0., f64::from(font_size)))
+            * Affine::skew(-20_f64.to_radians().tan(), 0.),
+    );
+    ctx.set_paint(REBECCA_PURPLE.with_alpha(0.5).into());
+    ctx.glyph_run(&font)
+        .font_size(font_size)
+        .glyph_transform(Affine::skew(20_f64.to_radians().tan(), 0.))
+        .hint(true)
+        .fill_glyphs(glyphs.into_iter());
+
+    check_ref(&ctx, "glyphs_skewed_unskewed");
```

Comment:
This rendering doesn't look right to me. If we're skewing each glyph individually, that should be a different operation to skewing the global box.
That is, this playground shows what I expect the rendering of this snapshot to be.
https://developer.mozilla.org/en-US/play?id=%2BlgWWSeKM7878QsZJVVId%2Bz0GMEv%2FqgYywTMjfjDnIW9K6nNfCtN5UPeH7%2FhgPFEQloDnLVzXIaKA7fG

(this is the same issue as my other comment)